_posts/2021-05-30-cloudnative-bot.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Using OpenFaas to run a fun twitter bot"
```

Comment:
Your bot is solving a problem that you saw, so I think benefit/solution would be a better title.

"I built a bot so that I would never miss a conference deadline." is a much more powerful and meaningful head line